keyboards/handwired/daishi/readme.md

```diff
@@ -0,0 +1,15 @@
+# Daishi
+
+![Daishi](http://kb.metamechs.com/wp-content/uploads/2019/03/IMG_20190325_113405-e1553610532162-1024x709.jpg)
+
+A compact battlecruiser using the CBC1 PCB. [More info on kb.metamechs.com](http://kb.metamechs.com/daishi/)
+
+Keyboard Maintainer: [Crokto](https://github.com/Croktopus/)  
+Hardware Supported: CBC1 PCB  
+Hardware Availability: [kb.metamechs.com](http://kb.metamechs.com/daishi/get-your-own-daishi/)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make daishi:default
```

Comment:
The make command needs to be updated to match the directory that has the source files.

```suggestion
    make handwired/daishi:default
```